ios-base/DroidKaigi 2020/en.lproj/Localizable.strings

```diff
@@ -4,7 +4,7 @@
 "DroidKaigi Description" = "DroidKaigi is a conference tailored for Android developers.\nIt's scheduled to take place on the 20 and 21 of February 2020.";
 "Access" = "Access";
 "Toc Building" = "Toc Building";
-"Maps" = "Maps";
+"Maps" = "Floor Maps";
```

Comment:
This is for `About`'s map strings.
This string represents such as `GoogleMap`.

Please create new Key 🙏

Reply:
@entaku19890818  If you have any questions please tell me. 🙏

Reply:
I had a misunderstanding!
I'll make a new key.

Reply:
fix done!